### ISSUE TITLE
LICENSE

### ISSUE THREAD

**Issue by bonyhardian** [amr-atef-afifi/BubbleShooter#1]
Hi bro,
What license for your code?
Can I use with Public Domain (CC0) or MIT?

Thanks

**Comment by amr-atef-afifi** [comment]
Yeah sure you can use it, modify it or whatever you want (as CC0 copyrights). Just a citation to our work would be a great appreciation.
